Automattic/jetpack
# AI Assistant: Add and use ImproveToolbarDropdownMenu in block toolbar

This PR adds and uses the `<ImproveToolbarDropdownMenu />` component.

Fixes #

## Proposed changes:

* AI Assistant: Add and use ImproveDropdown in block toolbar

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Create and use the AI Assistant block
* Confirm the Improve dropdown works as expected
  * `Make longer`, `Make shorter`,  `Summarize`
  * Confirm when it's editing the post title, the `Summarize` option is not listed

<img width="423" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/6c0b61ae-a6f4-46b3-9c71-0f75217b4652">
